### PR TITLE
Update tutorial instructions

### PR DESCRIPTION
**Pull request by mcflugen**
### Description

This pull request addresses #1526 by updating the instructions for installing the dependencies required to run the tutorial notebooks.

### Checklist - did you ...

- [x] Add a [news fragment](https://landlab.readthedocs.io/en/master/development/contribution/index.html#news-entries) file entry if necessary?
- [x] Add / update tests if necessary?
- [x] Add new / update outdated documentation?
- [x] All tests have passed?
- [x] Formatted code with black?
- [x] Removed lint reported by flake8?
- [x] Sucessful documentation built? (if documentation added or modified)

### CODE REVIEW

**Review by mdpiper — APPROVED** [review]

File: docs/source/user_guide/tutorials.rst [hunk 1]
```diff
@@ -39,32 +39,50 @@ Get the notebooks
...
-If you don't have a copy of the source code, you can run the
-`notebooks.py`_
+If you don't have a copy of the source code, you can run the `notebooks.py`_
 script to fetch the set of notebooks that matches your version of *landlab*.
 This can be done either by running,
 
 .. code-block:: bash
 
-   $ curl -L https://raw.githubusercontent.com/landlab/landlab/master/notebooks.py | python -
+   curl -L https://raw.githubusercontent.com/landlab/landlab/master/notebooks.py | python -
```
Comment (sebastien-lenard) [inline docs/source/user_guide/tutorials.rst:48]: I think comments should be added for this line, it's unclear how we can select the version which is spoken about L. 56.

**Review by sebastien-lenard — CHANGES_REQUESTED** [review]
An additional comment for the version may be necessary L.48.

File: docs/source/user_guide/tutorials.rst (hunk 1 above)
Comment (mcflugen) [reply docs/source/user_guide/tutorials.rst:48]: I agree. I'll add text that describes how this works.

Comment (mcflugen) [reply docs/source/user_guide/tutorials.rst:48]: @sebastien-lenard I've added an example that downloads a specific version of the notebooks.

**Comment by mcflugen** [comment]
I've also fixed the notebooks script to check what version of *landlab* is currently installed and, unless something different is given on the command line, download the notebooks for that version.

**Pull request merged**